Server/API/rooms/guess/postGuess.go

```diff
@@ -0,0 +1,51 @@
+package guess
+
+import (
+	"context"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/labstack/echo/v4"
+	roomsScripts "ishtaloo.io/DB/Scripts/rooms"
+	cookieUtils "ishtaloo.io/Utils/cookie"
+)
+
+func postGuess(c echo.Context) (err error) {
+	roomId := c.Param("roomId")
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	u, _ := cookieUtils.GetUserFromCookie(c)
+	usersInRoom, err := roomsScripts.GetUsersOfRoom(ctx, roomId)
```

Comment:
can't you add a query to check if the specific user exists in the room? why do we need the full list?

Reply:
I can make a query but I can also do it in go, so why do it in mongo? It feels a lot more straight forward to do it in go

Reply:
will do

Reply:
fixed